src/istep/edbgIstep.C

```diff
@@ -0,0 +1,572 @@
+  { 3,  18,                         "proc_sbe_scominit",                   EDBG_ISTEP_SBE},
+  { 3,  19,                              "proc_sbe_lpc",                   EDBG_ISTEP_SBE},
+  { 3,  20,                       "proc_sbe_fabricinit",                   EDBG_ISTEP_SBE},
+  { 3,  21,                     "proc_sbe_check_master",                   EDBG_ISTEP_SBE},
+  { 3,  22,                        "proc_sbe_mcs_setup",                   EDBG_ISTEP_SBE},
+  { 3,  23,                        "proc_sbe_select_ex",                   EDBG_ISTEP_SBE},
+  { 4,   1,                    "proc_hcd_cache_poweron",                   EDBG_ISTEP_SBE},
+  { 4,   2,                      "proc_hcd_cache_reset",                   EDBG_ISTEP_SBE},
+  { 4,   3,            "proc_hcd_cache_gptr_time_initf",                   EDBG_ISTEP_SBE},
+  { 4,   4,               "proc_hcd_cache_repair_initf",                   EDBG_ISTEP_SBE},
+  { 4,   5,                  "proc_hcd_cache_arrayinit",                   EDBG_ISTEP_SBE},
+  { 4,   6,                      "proc_hcd_cache_initf",                   EDBG_ISTEP_SBE},
+  { 4,   7,                "proc_hcd_cache_startclocks",                   EDBG_ISTEP_SBE},
+  { 4,   8,                   "proc_hcd_cache_scominit",                   EDBG_ISTEP_SBE},
+  { 4,   9,             "proc_hcd_cache_scom_customize",                   EDBG_ISTEP_SBE},
+  { 4,  10,           "proc_hcd_cache_ras_runtime_scom",                   EDBG_ISTEP_SBE},
+  { 4,  11,                     "proc_hcd_core_poweron",                   EDBG_ISTEP_SBE},
+  { 4,  12,               "proc_hcd_core_chiplet_reset",                   EDBG_ISTEP_SBE},
+  { 4,  13,             "proc_hcd_core_gptr_time_initf",                   EDBG_ISTEP_SBE},
+  { 4,  14,                "proc_hcd_core_repair_initf",                   EDBG_ISTEP_SBE},
+  { 4,  15,                   "proc_hcd_core_arrayinit",                   EDBG_ISTEP_SBE},
+  { 4,  16,                       "proc_hcd_core_initf",                   EDBG_ISTEP_SBE},
+  { 4,  17,                 "proc_hcd_core_startclocks",                   EDBG_ISTEP_SBE},
+  { 4,  18,                    "proc_hcd_core_scominit",                   EDBG_ISTEP_SBE},
+  { 4,  19,              "proc_hcd_core_scom_customize",                   EDBG_ISTEP_SBE},
+  { 4,  20,            "proc_hcd_core_ras_runtime_scom",                   EDBG_ISTEP_SBE},
+  { 5,   1,                  "proc_sbe_load_bootloader",                   EDBG_ISTEP_SBE},
+  { 5,   2,                   "proc_sbe_core_spr_setup",                   EDBG_ISTEP_SBE},
+  { 5,   3,                   "proc_sbe_instruct_start",                   EDBG_ISTEP_SBE},
+  { 6,   1,                           "host_bootloader",                   EDBG_ISTEP_HOST},
+  { 6,   2,                                "host_setup",                   EDBG_ISTEP_HOST},
+  { 6,   3,                         "host_istep_enable",                   EDBG_ISTEP_HOST},
```

Comment:
Need update in step 6 Name's not matching with IPL flow documents.  last step is 6.9